templates/docs/utilities/margin-collapse.md

```diff
@@ -14,6 +14,16 @@ Remove one or more margins of an element.
 View example of the margin collapse utility
 </a></div>
 
+# Responsive margin collapse
+
+<hr>
+
+Remove one or more margins of an element depending on the screen size.
+
+<div class="embedded-example"><a href="/docs/examples/utilities/margin-collapse-responsive/" class="js-example">
```

Comment:
I'm wondering about usefulness of this example as embedded in docs. It's quite long, size of the screen can't be changed in embedded example, etc.

As we discussed reducing it to bottom only, maybe lets to it first and see if the example will still be too much.